dot/parachain/prospective-parachains/inclusion_emulator.go

```diff
@@ -422,6 +440,66 @@ func skipUmpSignals(upwardMessages []parachaintypes.UpwardMessage) iter.Seq[para
 	}
 }
 
+func validateCommitments(
```

Comment:
Can this be a method of `CandidateCommitments`?

Reply:
We could but the errors would need to change too and I think there is no other place such validation occurs, so I see no problem keeping it here, wdyt?